### PR TITLE
Feature/184

### PR DESCRIPTION
**Pull request by milaendo**
Table needs to be discussed more (per Taylor), but the placeholder code is complete.

### CODE REVIEW

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+
+            <i class="seperator fas fa-arrow-right"></i>
+
+            <div class="table-wrapper">
+              <div class="d-flex head-wrapper">
+                <div class="table-head form-field">form FIELD</div>
+                <div class="table-head name-field">NAME</div>
+                <div class="table-head label-field">LABEL</div>
+                <div class="table-head type-field">TYPE</div>
+                <div class="table-head size-field">size</div>
```
Comment (velkymx) [inline resources/assets/js/designer/components/modals/modal-create-report-add.vue]: Why is size and index lowercase?

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+        </div>
+
+        <div class="form-group">
+          <label for="description">{{description}}</label>
+          <textarea class="form-control" id="description" rows="3"></textarea>
+        </div>
+
+        <div class="d-flex justify-content-between">
+          <div class="form-group">
+            <label for="type" v-model="type">{{type}}</label>
```
Comment (velkymx) [inline resources/assets/js/designer/components/modals/modal-create-report-add.vue]: Does the label need to be reactive?

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+      'typeSelectOptions':[
+        'View', 'Block'
+      ],
+      'dbConnection':"DB Connection",
+      'dbConnectionSelect': 'View',
+      'dbConnectionSelectOptions':[
+        'View', 'Block'
+      ],
+      list: [
+        {
```
Comment (velkymx) [inline resources/assets/js/designer/components/modals/modal-create-report-add.vue]: Please remove this dummy data. Do we know where the real data will come from? Is this something we can pull from the 3.0 tools?

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+   }
+  },
+  mounted() {
+    // Show our modal as soon as we're created
+    this.$refs.modal.show();
+  }
+};
+</script>
+<style lang="scss" scoped>
+.inline-input{
```
Comment (velkymx) [inline resources/assets/js/designer/components/modals/modal-create-report-add.vue]: Can some of this CSS be folded into the global styles sheet?

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue (hunk 1 above)
Comment (milaendo) [reply resources/assets/js/designer/components/modals/modal-create-report-add.vue]: copy and paste from zeplin, they have been changed.

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue (hunk 4 above)
Comment (milaendo) [reply resources/assets/js/designer/components/modals/modal-create-report-add.vue]: not that I know of, all of this is specific to this modal

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue (hunk 3 above)
Comment (milaendo) [reply resources/assets/js/designer/components/modals/modal-create-report-add.vue]: many of them are blank unless something has been assigned. I can just leave it blank for now.

File: resources/assets/js/designer/components/modals/modal-create-report-add.vue (hunk 2 above)
Comment (milaendo) [reply resources/assets/js/designer/components/modals/modal-create-report-add.vue]: thats just how I was asked to make the labels, can change if you want.

**Comment by velkymx** [comment]
I'm going to stop dev on this modal until we can get a clearer picture on how we want variables to work. @meadrat @tdondich

**Pull request closed without merging**